kratos/containers/distributed_vector_exporter.h

```diff
@@ -53,7 +50,7 @@ namespace Kratos
 
 /// Provides a DistributedVectorExporter which implements FEM assemble capabilities
 template<class TIndexType=std::size_t>
-class DistributedVectorExporter
+class DistributedVectorExporter final
```

Comment:
i see no strong reason for making this class underivable, nevertheless it is ok to me that if someone wants to derive will eventually have to remove the final so this gets communicated

Reply:
(same comment for the importer)